README.rst

```diff
@@ -0,0 +1,5 @@
+===============
+Macaroon Bakery
+===============
+
+A Python library for working with macaroons.
```

Comment:
A Python library for working with macaroons, compatible with the Go macaroon bakery package.